ops/mysql/Dockerfile

```diff
@@ -0,0 +1,18 @@
+ARG DIALECT=mysql:8.0
```

Comment:
Move this file under `.github`. There's no need to have it under the top level repository.